index.js

```diff
@@ -14,7 +14,11 @@ const debug = require('debug')('download-chromium')
 const cpr = promisify(require('cpr'))
 const mkdirp = promisify(require('mkdirp'))
 
-const get = url => new Promise(resolve => https.get(url, resolve))
+const proxy =
+  process.env.HTTPS_PROXY ||
+  process.env.npm_config_https_proxy ||
+  process.env.npm_config_proxy
+const get = url => new Promise(resolve => https.get({ url, proxy }, resolve))
```

Comment:
I can't seem to find either `url` or `proxy` in [the node docs for https](https://nodejs.org/api/https.html#https_https_get_options_callback). This change appears to have some regressions (https://github.com/juliangruber/download-chromium/issues/9).

Reply:
You're right! I'm fixing this.